test/config.ts

```diff
@@ -0,0 +1,4 @@
+export const TEST_RINKEBY_PUBLIC_ADDRESS = process.env.TEST_RINKEBY_PUBLIC_ADDRESS || '';
+export const TEST_RINKEBY_PRIVATE_KEY = process.env.TEST_RINKEBY_PRIVATE_KEY || '';
+export const TEST_RINKEBY_RPC_URL = process.env.TEST_RINKEBY_RPC_URL || '';
+export const TEST_RINKEBY_CHAIN_ID = 4;
```

Comment:
Do we have enough Rinkeby ETH to have this test running on every CI run?

Reply:
yes

Reply:
Should we run this 24/7 legitimately on Kovan? That way we can make actual trades against liquidity.

Or is Kovan too fast to be a good environment?